docs/install_guides/installing-openvino-yum.md

```diff
@@ -1,10 +1,210 @@
+
+## System Requirements
+
+The complete list of supported hardware is available in the [Release Notes](https://software.intel.com/content/www/us/en/develop/articles/openvino-relnotes.html).
+
+**Operating systems**
+
+- Red Hat Enterprise Linux 8, 64-bit
+
+**Software**
+
+- [CMake 3.13 or higher, 64-bit](https://cmake.org/download/)
+- GCC 8.2.0
+- [Python 3.7 - 3.10, 64-bit](https://www.python.org/downloads/windows/)
+
+## Install OpenVINO Runtime
+
+### Step 1: Set Up the Repository
+
+1. Create the YUM repo file in the `/tmp` directory as a normal user:
+   ```
+   tee > /tmp/openvino-2022.repo << EOF
+   [OpenVINO]
+   name=Intel(R) Distribution of OpenVINO 2022
+   baseurl=https://yum.repos.intel.com/openvino/2022
+   enabled=1
+   gpgcheck=1
+   repo_gpgcheck=1
+   gpgkey=https://yum.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+   EOF
+   ```
+2.	Move the new openvino-2022.repo file to the YUM configuration directory `/etc/yum.repos.d`:
+   ```sh
+   sudo mv /tmp/openvino-2022.repo /etc/yum.repos.d
+   ```
+3.	Verify that the new repo is properly setup by running the following command:
+   ```sh
+   yum repolist | grep -i openvino
+   ```
+    You will see the available list of packages.
+
+
+To list available OpenVINO packages, use the following command:
+
+@sphinxdirective
+
+   .. code-block:: sh
+
+      yum list 'openvino*'
+
+@endsphinxdirective
+
+### Step 2: Install OpenVINO Runtime Using the YUM Package Manager
+
+OpenVINO™ Runtime will be installed in: `/opt/intel/openvino_<VERSION>.<UPDATE>.<PATCH>`
+
+A symlink will be created: `/opt/intel/openvino_<VERSION>`
+
+#### Install OpenVINO Runtime
+
+@sphinxdirective
+
+.. tab:: The Latest Version
+
+   Run the following command:
+
+   .. code-block:: sh
+
+      sudo yum install openvino
+
+.. tab::  A Specific Version
+
+   Run the following command:
+
+   .. code-block:: sh
+
+      sudo yum install openvino-<VERSION>.<UPDATE>.<PATCH>
+
+   For example:
+
+   .. code-block:: sh
+
+      sudo yum install openvino-2022.3.0
+
+@endsphinxdirective
+
+
+#### Check for Installed Packages and Version
+
+Run the following command:
+
+@sphinxdirective
+
+   .. code-block:: sh
+
+      yum list installed 'openvino*'
+
+@endsphinxdirective
+
+
+#### Uninstall OpenVINO Runtime
+
+@sphinxdirective
+
+.. tab:: The Latest Version
+
+   Run the following command:
+
+   .. code-block:: sh
+
+      sudo yum autoremove openvino
+
+
+.. tab::  A Specific Version
+
+   Run the following command:
+
+   .. code-block:: sh
+
+      sudo yum autoremove openvino-<VERSION>.<UPDATE>.<PATCH>
+
+   For example:
+
+   .. code-block:: sh
+
+      sudo yum autoremove openvino-2022.3.0
+
+@endsphinxdirective
+
+
+### Step 3 (Optional): Install OpenCV from YUM
```

Comment:
1. Remove OpenCV section
2. Add build samples section as it was written in debian document

Reply:
Removed with https://github.com/openvinotoolkit/openvino/pull/14492/commits/bce5a845f8329dbc9b09da05817f04efb5acf19d